packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -441,7 +441,7 @@ export class DeltaManager
             });
 
         this._inbound.on("error", (error) => {
-            this.close(CreateProcessingError(error, this.lastMessage));
+            this.close(CreateProcessingError(error, "DeltaManager Inbound Error", this.lastMessage));
```

Comment:
what is the expected format for errorCode, i was thinking more like eventName which i believe is Pascal case, rather than separate words

Reply:
I don't really care. I originally said pascaleCase but upon further reflection I don't really see a problem with words which is more readable.  There's tons of sentences passed as message today which will be moved to error code, and might be nice to keep them.

Happy to flip these new ones though.